src/CPLEX.jl

```diff
@@ -84,6 +85,11 @@ module CPLEX
     #        cplex_model
 
     using Compat
+    
+    if VERSION >= v"0.7.0-DEV.3382"
+        using SparseArrays
+        using LinearAlgebra
+    end
```

Comment:
We could also use `Compat.XXX`

Reply:
given the guideline update on [discourse](https://discourse.julialang.org/t/jump-and-julia-0-7-1-0/12964), I think we can remove this all-together and use 1.0-compatible features only
So remove the version check and no Compat

Reply:
@matbesancon, but those are needed for 0.7 and above.